### PR TITLE
Classes examples

### PR DESCRIPTION
**Pull request by WChrisK**
Note: Thought there already was a pull request for this, my bad. It has the new updates anyways.

### CODE REVIEW

File: examples/pylint/E0202_method_hidden.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Example(object):
```
Comment (david-yz-liu) [inline examples/pylint/E0202_method_hidden.py]: You can disable this check. I'd prefer an error on masking a method (not for right now, just in general).

File: examples/pylint/E0203_access_member_before_definition.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class MyClass:
+	def __init__(self):
```
Comment (david-yz-liu) [inline examples/pylint/E0203_access_member_before_definition.py]: Please use 4 spaces. Not sure if you're using tabs or not.

File: examples/pylint/E0213_no_self_argument.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class MyClass:
```
Comment (david-yz-liu) [inline examples/pylint/E0213_no_self_argument.py]: Question: how is this different from the previous error? Will this error be emitted if the first argument is something other than `self`?

File: examples/pylint/E0241_duplicate_bases.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class A:
```
Comment (david-yz-liu) [inline examples/pylint/E0241_duplicate_bases.py]: Disable this check

File: examples/pylint/E0704_misplaced_bare_raise.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+class CustomException(Exception):
```
Comment (david-yz-liu) [inline examples/pylint/E0704_misplaced_bare_raise.py]: This is interesting. We won't be strict on this in 148, so please disable the check.

File: examples/pylint/W0211_bad_static_method_argument.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+class C:
```
Comment (david-yz-liu) [inline examples/pylint/W0211_bad_static_method_argument.py]: Disable this check.

File: examples/pylint/W0711_binary_op_exception.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+class MyDoubleException(Exception):
+        pass
+        
+def binary_capture():
+        try:
+                # Not caught, 'or' doesn't do what you think.
+                # Need to do: except (MyException, MyDoubleException):
+                raise MyDoubleException()
+        except MyException or MyDoubleException:
```
Comment (david-yz-liu) [inline examples/pylint/W0711_binary_op_exception.py]: I know you mentioned this one already, but cool!

**Comment by david-yz-liu** [comment]
@WChrisK A reminder that this pull request has some outstanding comments.

File: examples/pylint/E0213_no_self_argument.py (hunk 3 above)
Comment (WChrisK) [inline examples/pylint/E0213_no_self_argument.py]: As a note (even though its in the commit), this was one that I forgot to change in the conversion (pylint considers something like _def method(asd)_ to be an error, _asd_ should be _self_, even though its not a Python error)

File: examples/pylint/E0302_unexpected_special_method_signature.py [hunk 8]
```diff
@@ -0,0 +1,11 @@
+class A:
+	def __init__(self):
```
Comment (nigef) [inline examples/pylint/E0302_unexpected_special_method_signature.py]: 👆 for all examples👇

correct me if i'm wrong; i believe we are meant to use short descriptive docstrings (with double quotes) in the classes, methods, and functions.

Comment (david-yz-liu) [inline examples/pylint/E0302_unexpected_special_method_signature.py]: I haven't decided what I want for docstrings for these examples, so for now I will accept this.

@WChrisK seems like your examples use tabs. Please convert indentation to 4 spaces.

**Comment by david-yz-liu** [comment]
Thanks, @WChrisK!

**Pull request merged**